interfaces/Common/IERC20.sol

```diff
@@ -1,20 +1,20 @@
-// SPDX-License-Identifier: MIT

```

Comment:
Not sure why it's picked up all of these files as new, do you know what the linter changed in them?

Reply:
End of line sequences were changed from CRLF to LF for some files. Would assume Prettier defaults to LF, CRLF is mostly used by Windows if i'm not mistaken.